lms/static/sass/shared/_course_object.scss

```diff
@@ -267,3 +267,23 @@
     }
   }
 }
+
+.xmodule_licensable {
+  @include transition(opacity .25s ease-in-out);
```

Comment:
NIT - I would use one of our tmg sass variables here instead of .25s specifically.